src/algorithms/post/election_post.md

```diff
@@ -40,32 +40,42 @@ The epoch time is divided into three non-overlapping portions: (1) checking sect
+1. **(sample randomness)**
+The miner draws a randomness ticket from the randomness chain from a given epoch SPC.post_lookback back and concats it with the minerID, and epoch for use as post_randomness (this is as it used to happen previously, except the PoSt_lookback is now also the EC randomness_lookback, which is small here):
     - `post_randomness = VRF_miner(ChainRandomness(currentBlockHeight - SPC.post_lookback))`
-    We use the VRF to generate a unique randomness for the minerId, in this way no adversary that could simulate the miner storage can check if the victim miner has won or not.
-1. **(select eligible sectors)** The miner calls `GenerateCandidates` from proofs with their non-faulted (declared or detected) sectors, meaning those in their `proving set` (from the `StorageMinerActor`) along with a chosen number of `PartialTickets` (`sectorsSampled*eligibleSectors`).
+
+2. **(select eligible sectors)**
+The miner calls `GenerateCandidates` from proofs with their non-faulted (declared or detected) sectors, meaning those in their `proving set` (from the `StorageMinerActor`) along with a chosen number of `PartialTickets` (`sectorsSampled*eligibleSectors`).
     - Select a subset of sectors of size `sectorsSampled*minerSectors` [details omitted]
+
     Note also that even with `challengeTicketNum == numSectors`, this process may not sample all of a miner’s sectors in any given epoch, due to collisions in the pseudorandom sector ID selection process.
-1. **(generate Partial Ticket(s))** For each selected sector
+
+3. **(generate Partial Ticket(s))** for each selected sector
     - first, generate K `PoStChallenges` (C_i), sampled at random from the chosen sector.
     There will be a chosen challenge-range size (a power of 2), called `ChallengeRangeSize`. Challenge ranges must be subtree-aligned and thus divide the sector into fixed-size data blocks. So challenge ranges can be indexed because allowable challenge ranges do not overlap. C_i is selected from the valid indexes into the challenge ranges.
     `C_i = HashToBlockNumber(post_randomness || S_j || i)`
     C_i corresponds to a given index of a known-sized data block (the sector)
     - for each C_i, miner reads the specified range of data from disk, which is of size `ChallengeRangeSize : C_i_output`
     - Miner generates a PartialTicket using all the PoSt witnesses
     - `PartialTicket = H(post_randomness || minerID ||S_ j || C_1_Output || … || C_K_Output)`
-1. **(check Challenge Ticket(s) for winners)** Given a returned ChallengeTicket, miner checks it is a winning ticket. Specifically, they do the following:
+
+4. **(check Challenge Ticket(s) for winners)**
+Given a returned PartialTicket, miner checks it is a winning ticket. Specifically, they do the following:
     - `ChallengeTicket = Finalize(PartialTicket).  = H(ChallengeTicket) / 2^len(H)`
```

Comment:
I think this line may have become garbled.

`ChallengeTicket = Finalize(PartialTicket)` makes sense, but the rest of the line is unclear. To what is the expression `H(ChallengeTicket) / 2^len(H)` being assigned?

Apart from some simple confusion in what was written, I think there may also still be a terminology mismatch. I'm going to make a guess and rewrite this with the added term I think is missing:

```
FinalTicket = Finalize(PartialTicket)
ChallengeTicket = FinalTicket / 2^len(H)
```

In order to be consistent with current implementation (as we decided to do), we need for `Finalize(…)` to correspond to `H(…)`. In order to preserve your meaning for `ChallengeTicket`, we can just call the result of finalization `FinalTicket`.

Reply:
I thought the partialTicket already corresponded to `H(...)` and Finalization was the act of doing `/2^len(H)` yielding a `ChallengeTicket`

If we can, let's try to only name two values: that yielded by the proofs library and stored on chain (what I understsand as the partial ticket, what you call a finalTicket above) and that which is checked in LE (what I call a ChallengeTicket but we can call it something else)